serenata_toolbox/chamber_of_deputies/dataset.py

```diff
@@ -137,3 +132,6 @@ def __translate_file(self, csv_path):
                     encoding='utf-8')
 
         return output_file_path
+
+    def __parse_float(self, string):
```

Comment:
There is any special motive to have two trailing underscore here? 

Could we use just one to say that it's just to be used internally and not a special python method.